packages/db/src/schema/page.ts

```diff
@@ -54,7 +54,7 @@ export const insertPageSchemaWithMonitors = insertPageSchema.extend({
   customDomain: z.string().optional().default(""),
   monitors: z.array(z.number()).optional(),
   workspaceSlug: z.string().optional(),
-  slug: z.string().toLowerCase(),
+  slug: z.string().toLowerCase().min(1),
```

Comment:
We [overwrite](https://zod.dev/?id=extend) the `insertPageSchema` and it's defined slug with regex and `min(3)` a few lines above. Let's remove that line entirely and add the `.toLowerCase()` to the `insertPageSchema.slug`.

Reply:
Or use the same `regex` and `min(3)` as well here.